packages/cli/commons/src/platform/authenticatedClient.ts

```diff
@@ -39,12 +41,47 @@ export class AuthenticatedClient {
     if (!Object.keys(global).includes('FormData')) {
       Object.assign(global, {FormData});
     }
+
     return new PlatformClient({
       globalRequestSettings,
       environment: castEnvironmentToPlatformClient(resolvedConfig.environment),
       region: resolvedConfig.region,
       organizationId: resolvedConfig.organization,
       accessToken: resolvedConfig.accessToken!,
+      responseHandlers: [
+        {
+          canProcess: (response: Response): boolean => {
+            return response.ok;
+          },
+          process: async <T>(response: Response) => {
+            const content = await response.text();
+            return (
+              JSON.parse as (
+                text: string,
+                reviver?: (
+                  key: string,
+                  value: any,
+                  context: {source: any}
+                ) => any | undefined
+              ) => any
+            )(content, (key, value, context) => {
+              if (
+                key === 'rowid' &&
+                typeof value === 'number' &&
+                !Number.isSafeInteger(value)
+              ) {
+                return context.source;
+              }
+              return value;
+            });
+          },
+        },
+        ResponseHandlers.badGateway,
```

Comment:
So we are "copying" the handlers from platform client as a "snapshot" of what it does today ?

Which means these will drift over time....

Reply:
True, there could be additional default response handlers in the future. I'll see if I can do something better than that.

Reply:
Why not put the handler in the platform-client code?

Reply:
I'm hesitant to do that because it would technically be introducing a breaking change in the platform-client...